src/lib/validation/utils/helpers.js

```diff
@@ -16,6 +16,8 @@ function shouldValidate (input) {
     isRequired ||
     input.hasAttribute('pattern') ||
     input.getAttribute('type') === 'email' ||
+    input.getAttribute('name') === 'r_startdate' ||
+    input.getAttribute('name') === 'r_enddate' ||
```

Comment:
see the other comment. let's change this into:

```suggestion
    input.hasAttribute('data-ad-id')
```

and always validate date fields.

Reply:
done